### PR TITLE
Clear active tool on escape

### LINKED ISSUE #267: Pressing escape after selecting a shape should revert back to selection

### PR DESCRIPTION
**Pull request by giovannigiordano**
Fixes #267

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -147,13 +147,20 @@ export class App extends React.Component<{}, AppState> {
   };
 
   private onKeyDown = (event: KeyboardEvent) => {
-    if (isInputLike(event.target)) return;
-
     if (event.key === KEYS.ESCAPE) {
       clearSelection(elements);
       this.forceUpdate();
+      console.log('selection')
```
Comment (Fausto95) [inline src/index.tsx]: you forgot to remove this log

Comment (dwelle) [reply src/index.tsx]: We should lint against console logs.

**Comment by dwelle** [comment]
LGMT. Thanks!

**Comment by dwelle** [comment]
Mhm, there were 2 other lint issues (quotes & semicolon). I'm wondering why it was not caught by linter/prettier.

**Pull request merged**